current_spring2026/evaluation/eval_graphrag.py

```
@@ -0,0 +1,276 @@
+                row[f"graph_recall_{k}"]    = graph_metrics[f"recall_{k}"]
+                row[f"graph_precision_{k}"] = graph_metrics[f"precision_{k}"]
+                row[f"hit_delta_{k}"]       = graph_metrics[f"hit_{k}"]    - dense_metrics[f"hit_{k}"]
+                row[f"recall_delta_{k}"]    = round(graph_metrics[f"recall_{k}"]    - dense_metrics[f"recall_{k}"], 4)
+                row[f"precision_delta_{k}"] = round(graph_metrics[f"precision_{k}"] - dense_metrics[f"precision_{k}"], 4)
+
+            for k in [10, 30]:
+                d = dense_metrics[f"hit_{k}"]
+                g = graph_metrics[f"hit_{k}"]
+                print(f"  hit@{k}: dense={d}  graph={g}  Δ={g-d:+d}")
+
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)